doc/source/quickstart.rst

```diff
@@ -145,6 +159,23 @@ You have the possibility to configure on any command the PATH to the SDK, NDK an
 - :code:`--ndk-api VERSION` as an equivalent of `$NDKAPI`
 - :code:`--ndk-version VERSION` as an equivalent of `$ANDROIDNDKVER`
 
+Special case: SDK/NDK on 32bit-systems
```

Comment:
I'd be happy to just ditch this doc, I bet there are all sorts of problems users would have if they tried this, and it's so old that we don't remotely support it.

Reply:
ok, I removed it along with another small remark about 32-bit archlinux :+1: I'm all for simplification